src/Tizen.NUI/src/public/GLWindow.cs

```diff
@@ -392,13 +401,18 @@ public GLWindow.GLWindowOrientation GetCurrentOrientation()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public void SetAvailableOrientations(List<GLWindow.GLWindowOrientation> orientations)
         {
+            if (orientations == null)
+                throw new ArgumentNullException(nameof(orientations));  // To fix CA1062 issue.
```

Comment:
could you add exception comment ? 
<exception cref=xxx> xxx </exception>
(example) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69

Reply:
Done
About warning CA1062, follow the comment of Mr. YeongJong Lee, remove it from current patch.